src/Symfony/Component/Validator/Constraints/Json.php

```diff
@@ -11,32 +11,28 @@
 
 namespace Symfony\Component\Validator\Constraints;
 
-use Symfony\Component\Validator\Constraint;
-
 /**
  * Validates that a value has valid JSON syntax.
  *
  * @author Imad ZAIRIG <imadzairig@gmail.com>
  */
 #[\Attribute(\Attribute::TARGET_PROPERTY | \Attribute::TARGET_METHOD | \Attribute::IS_REPEATABLE)]
-class Json extends Constraint
+class Json extends AbstractFormat
 {
     public const INVALID_JSON_ERROR = '0789c8ad-2d2b-49a4-8356-e2ce63998504';
 
     protected const ERROR_NAMES = [
         self::INVALID_JSON_ERROR => 'INVALID_JSON_ERROR',
     ];
 
-    public string $message = 'This value should be valid JSON.';
```

Comment:
changing the default message of this constraint will break any project relying on translating this message.